apps/wellknown/views/application.py

```diff
@@ -1,10 +1,10 @@
-from ...dot_ext.models import Application
-from ..serializers import ApplicationListSerializer
 from django.conf import settings
 from rest_framework.generics import ListAPIView
 from rest_framework.pagination import PageNumberPagination
 from rest_framework.permissions import AllowAny
 from rest_framework.renderers import JSONRenderer
+from ...dot_ext.models import Application, ApplicationLabel
```

Comment:
Because this is a dependency on a different module (app) the whole path should be declared `app.dot_ext.models`